### PR TITLE
Added function to toggle resuming of last song when app starts

### PR DESCRIPTION
**Pull request by NNowakowski**
Implemented option to toggle the resume behaviour as requested in #161

### CODE REVIEW

**Comment by ashirbhallalevine** [comment]
@NNowakowski Which toggle option is this and should it be available if I have auto-updates turned on?

**Comment by NNowakowski** [comment]
> @NNowakowski Which toggle option is this and should it be available if I have auto-updates turned on?

This request wasn't merged yet. Once it gets added to the codebase it should be available with the next update following afterwards.

**Comment by ashirbhallalevine** [comment]
> > @NNowakowski Which toggle option is this and should it be available if I have auto-updates turned on?
> 
> This request wasn't merged yet. Once it gets added to the codebase it should be available with the next update following afterwards.

Ah, I see. My bad, I'm fairly new to GitHub. And now I also see the "files changed" tab which explains where/how the change will be implemented. Thank you for writing up the changes and responding to my question so quickly!

**Comment by NNowakowski** [comment]
> > > @NNowakowski Which toggle option is this and should it be available if I have auto-updates turned on?
> > 
> > 
> > This request wasn't merged yet. Once it gets added to the codebase it should be available with the next update following afterwards.
> 
> Ah, I see. My bad, I'm fairly new to GitHub. And now I also see the "files changed" tab which explains where/how the change will be implemented. Thank you for writing up the changes and responding to my question so quickly!

No problem. It's a feature I wanted to see aswell so seeing your request gave me the push I needed to implement this :p

File: index.js [hunk 1]
```diff
@@ -111,7 +111,8 @@ function createMainWindow() {
 		win.maximize();
 	}
 
-	win.webContents.loadURL(config.get("url"));
+	const urlToLoad = config.get("resumeOnStart") ? config.get("url") : "https://music.youtube.com";
```
Comment (th-ch) [inline index.js:114]: nit: root url could be imported from default config

Comment (th-ch) [reply index.js:114]: Small typo: option is named `resumeSongOnStart` in other places and not `resumeOnStart`

**Review by th-ch — APPROVED** [review]
Just 2 small fixes, I will make a quick follow up to address them - thanks for the contribution, merging!

**Pull request merged**